variables.yml

```diff
@@ -48,15 +46,15 @@ dependencies:
       version: 1.0.4
     asset_transfer_api:
       name: '@substrate/asset-transfer-api'
-      version: 0.6.0
-      polkadot_js_api_version: v15.8.1 # Check changelog for pjs version bumps
+      version: 0.6.1
+      polkadot_js_api_version: v15.9.2 # Check changelog for pjs version bumps
```

Comment:
I'm assuming you checked the links that are anchored to specific lines for this change? In the `polkadot-docs/develop/toolkit/interoperability/asset-transfer-api/reference.md` file

Reply:
Yes I did

Reply:
Beautiful! Thanks! 🙂